logic/transaction_pool.js

```diff
@@ -464,13 +483,9 @@ TransactionPool.prototype.reindexQueues = function() {
 
 /**
  * Processes the next bundle of transactions and add them to the queued index.
- * @implements {getBundledTransactionList}
- * @implements {removeBundledTransaction}
- * @implements {processVerifyTransaction}
- * @implements {removeUnconfirmedTransaction}
- * @implements {queueTransaction}
- * @param {function} cb - Callback function.
- * @return {setImmediateCallback} err | cb
+ *
+ * @param {function} cb - Callback function
+ * @returns {setImmediateCallback} err | cb
```

Comment:
This is different to `err, transactions` above. Which do we want? Does it even make sense?

Reply:
As mentioned above, I think after standardization it makes sense to revisit the docs and complete/ correct them.
I'm not sure this is part of this PR, but we could leave `@todo`s for everything that should be looked over at again.